src/main/app/src/components/valintaperusteet/Valintakokeet.tsx

```diff
@@ -129,93 +125,102 @@ export const Valintakokeet = ({
-              id={`${toId(localize(nimi))}`}
-              elevation={0}
-              style={{
-                backgroundColor: colors.grey,
-                padding: '15px',
-                marginBottom: '20px',
-              }}>
-              <CardContent>
-                <Typography variant="body1">{localizedTyyppi}</Typography>
-                <Typography className={classes.valintakoeHeader} variant="h4">
-                  {localize(nimi)}
-                </Typography>
-                {!_.isEmpty(tietoja) && <LocalizedHTML data={tietoja!} />}
-                {vahimmaispisteet && (
-                  <>
-                    <SubHeading>
-                      {t('valintaperuste.alin-hyvaksytty-pistemaara')}
-                    </SubHeading>
-                    <Typography variant="body1">{vahimmaispisteet}</Typography>
-                  </>
-                )}
-                {!_.isEmpty(ohjeetEnnakkovalmistautumiseen) && (
-                  <>
-                    <SubHeading>
-                      {t('valintaperuste.valmistautumisohjeet-hakijalle')}
-                    </SubHeading>
-                    <LocalizedHTML data={ohjeetEnnakkovalmistautumiseen!} />
-                  </>
-                )}
-                {!_.isEmpty(ohjeetErityisjarjestelyihin) && (
-                  <>
-                    <SubHeading>
-                      {t('valintaperuste.ohjeet-erityisjarjestelyihin')}
-                    </SubHeading>
-                    <LocalizedHTML data={ohjeetErityisjarjestelyihin!} />
-                  </>
-                )}
-                {tilaisuudet.length > 0 && (
-                  <Box mt={2}>
-                    <Accordion
-                      noColors
-                      ContentWrapper={'div' as any}
-                      items={tilaisuudet.map((tilaisuus, index) => ({
-                        title: `${t('valintaperuste.tilaisuus')} ${index + 1}`,
-                        content: (
-                          <TilaisuusComponent index={index} tilaisuus={tilaisuus} />
-                        ),
-                      }))}
-                    />
-                  </Box>
-                )}
-              </CardContent>
-            </Card>
-          </div>
-        );
-      })}
+            return (
+              <div key={`valintakoe-${index}`}>
+                <Card
+                  id={`${toId(localize(nimi))}`}
+                  elevation={0}
+                  style={{
+                    backgroundColor: colors.grey,
+                    padding: '15px',
+                    marginBottom: '20px',
+                  }}>
+                  <CardContent>
+                    <Heading variant="body1">
+                      {localizedTyyppi}
+                      <hr style={{ borderStyle: 'none', margin: 0 }} />
+                      <Box style={{ fontSize: '20px', fontWeight: 'bold' }}>
+                        {localize(nimi)}
+                      </Box>
```

Comment:
Tämä ei ole semanttisesti oikein hyvä sisältö tälle headingille, `div` ja `hr` headingin sisällä on käytännössä invalidia syntaksia. Jotenkin tälleen olis ehkä parempi. `theme`:n saaa `useTheme`:lla.
```suggestion
                      <span style={{...theme.typography.body1, clear: 'right'}}>{localizedTyyppi}</span>
                      <span style={theme.typography.h4}>{localize(nimi)}</span>
```